mappings/net/minecraft/data/server/LootTablesProvider.mapping

```diff
@@ -2,9 +2,16 @@ CLASS net/minecraft/class_1285 net/minecraft/data/server/LootTablesProvider
 	FIELD field_26612 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_26613 GSON Lcom/google/gson/Gson;
 	FIELD field_26614 cache Lcom/google/common/cache/LoadingCache;
+	FIELD field_26615 json Ljava/io/File;
+	METHOD <init> (Ljava/io/File;)V
+		ARG 1 lootTableJson
 	METHOD method_28401 reload ()V
 	METHOD method_28403 getLootTable (Lnet/minecraft/class_4523;)Lnet/minecraft/class_5515;
 		ARG 1 identifier
-	CLASS class_5522
-		METHOD load load (Ljava/lang/Object;)Ljava/lang/Object;
+	CLASS class_5522 Cache
```

Comment:
this isnt the method to fix these. these should be fixed in intermediary.

Reply:
Wot? Why? How?

Reply:
OK there's nothing wrong with this fix. It unmaps the method in Yarn. There's nothing actually wrong with the intermediaries. Please look at it again.

Reply:
I'm gonna map it again now for complentionism's sake (thie file will be 100% complete after I do) but I hope you see the point.